### ISSUE TITLE
Changing category bar on mobile

### ISSUE THREAD

**Issue by georgeteo** [uchicago-sg/caravel#108]

**Comment by fatlotus** [comment]
Wait! This is still broken methinks

**Comment by georgeteo** [comment]
Oops. What was the issue here again?

**Comment by fatlotus** [comment]
<img width="1200" alt="screen shot 2015-11-15 at 12 45 34 pm" src="https://cloud.githubusercontent.com/assets/14364/11170343/d238ddae-8b96-11e5-979e-63d4f122f50c.png">
<img width="907" alt="screen shot 2015-11-15 at 12 45 38 pm" src="https://cloud.githubusercontent.com/assets/14364/11170344/d239e1e0-8b96-11e5-8679-beacb9c67312.png">

**Comment by fatlotus** [comment]
turns out it's only _mostly_ responsive

![tumblr_loww57mhwi1qasdqt](https://cloud.githubusercontent.com/assets/14364/11170345/f4a8deac-8b96-11e5-94a5-0d7d60cd0529.jpg)

**Comment by georgeteo** [comment]
Issue is the category bar or the padding aroudn "new listing, ..."

**Comment by fatlotus** [comment]
I guess both... the whole thing has some wonk.
dammit

**Comment by georgeteo** [comment]
Padding is my bad. 

Thoughts on a good way of dealing with category overflow when the width of the page is not quite short enough to do to the full mobile view?

Some thoughts:
1. Truncate category names
2. Do the mobile view earlier
3. Two rows

**Comment by fatlotus** [comment]
Yeah, I honestly don't know. The mobile view is already a little weird — so maybe (3)? But then we'd have three regimes rather than just two.

**Comment by tranv94** [comment]
Thoughts on using a dropdown menu like amazon for desktop?

<img width="432" alt="screen shot 2015-11-30 at 1 37 39 am" src="https://cloud.githubusercontent.com/assets/6442177/11465945/099128ca-9703-11e5-9284-51cc42ef5682.png">

And for mobile, maybe a dedicated page for categories via button?

**Comment by georgeteo** [comment]
Yeah I was thinking like a sidebar that comes out when a button is pressed. Was going to do it as part of the polymer migration when I have some time. (Thinking winter break)

**Comment by tranv94** [comment]
I'm on winter break now with a surplus of free time, are you working on a separate repository for the polymer integration? I would love to help :)

**Comment by fatlotus** [comment]
Right now, I'm focused mostly on switching backends (see PR), but that's definitely something we're considering! Talk to @georgeteo

**Comment by georgeteo** [comment]
@tranv94: Hi! Let's chat if you're still interested in fiddling around with front end changes. We're thinking react + semantic UI. (I've FB messaged you.)
